frontend/.angular-cli.json

```diff
@@ -19,7 +19,6 @@
       "testTsconfig": "tsconfig.spec.json",
       "prefix": "volontulo",
       "styles": [
-        "assets/css/bootstrap-glyphicons.css",
```

Comment:
if we are not going to use that style, shouldn't we also remove file itself from repository?